### ISSUE TITLE
when a user has no name, it deadlocks the network

### ISSUE THREAD

**Issue by telephon** [muellmusik/Utopia#15]
where two computers have the same name, or `nil`, there is a constant flow of messages:

`Peer nil rejoined the Utopia`

**Comment by telephon** [comment]
Somewhere in

```
add {|peer|
		peer = peer.as(Peer);
		dict[peer.name] = peer;
		peer.addDependant(this);
		this.changed(\add, peer)
	}
```

we may have to check if it is duplicate with ref to IP address

**Comment by muellmusik** [comment]
Yes, I think that's sensible. Probably it should just use a kind of filename increment scheme (e.g. telephon-2) and post a warning.

**Comment by telephon** [comment]
Yes, if it warns once, this would be enough.
